shell-definition.yaml

```diff
@@ -1,9 +1,9 @@
 tosca_definitions_version: tosca_simple_yaml_1_0
 
 metadata:
-  template_name: Openstack Shell 2G
+  template_name: OSF-OpenStack-Cloud-Provider-Shell-2G
```

Comment:
This is what the user would see on the portal in the Shells Menu, I believe we need to remove `-`